modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/asset/model/ObjectEntryFolderAssetRendererFactory.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.web.internal.asset.model;
+
+import com.liferay.asset.kernel.model.AssetRenderer;
+import com.liferay.asset.kernel.model.AssetRendererFactory;
+import com.liferay.asset.kernel.model.BaseAssetRendererFactory;
+import com.liferay.object.constants.ObjectPortletKeys;
+import com.liferay.object.model.ObjectEntryFolder;
+import com.liferay.object.service.ObjectEntryFolderService;
+import com.liferay.portal.kernel.exception.PortalException;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Mikel Lorza
+ */
+@Component(
+	property = "javax.portlet.name=" + ObjectPortletKeys.OBJECT_DEFINITIONS,
+	service = AssetRendererFactory.class
+)
+public class ObjectEntryFolderAssetRendererFactory
+	extends BaseAssetRendererFactory<ObjectEntryFolder> {
+
+	public static final String TYPE = "objectEntryFolder";
```

Comment:
I've a doubt, I understand we can put anything here but I've seen that the most common pattern is using lowercase words separated with underscore, should we use this pattern?

Reply:
ok, I will improve it.

thanks